vignettes/example1.Rmd

```diff
@@ -91,20 +90,21 @@ You can see the sampling algorithm being run.
 
 ### Extracting the posterior
 
-Once it is done, let us extract the parameters (*i.e.*, the coefficients) of the model.
+Once it is done, let us extract the parameters (*i.e.*, coefficients) of the model.
 
 ```{r message=FALSE, warning=FALSE, eval=FALSE}
 posteriors <- insight::get_parameters(model)
 
 head(posteriors)  # Show the first 6 rows
 ```
+
 ```{r message=FALSE, warning=FALSE, echo=FALSE}
 posteriors <- insight::get_parameters(model)
 
 head(posteriors)  # Show the first 6 rows
 ```
 
-As we can see, the parameters take the form of a long dataframe with two columns, corresponding to the **intercept** and the effect of **Petal.Length**. These columns contain the **posterior distributions** of these two parameters, *i.e.*, a set of different plausible values.
+As we can see, the parameters take the form of a wide dataframe with two columns, corresponding to the intercept and the effect of `Petal.Length`. These columns contain the **posterior distributions** of these two parameters. In simple terms, the posterior distribution is a set of different plausible values for each parameter.
```

Comment:
actually here the dataframe is "long" (i.e. more rows than columns) ☺️ 

I would also add batsticks around `intercept` as it refers to the parameter name

Reply:
My understanding is that long refers to the format of:

`MEASURE,VALUE` while wide basically means `ID, VAR1, VAR2` with values contained in columns?

I could be wrong, but I don't think it's about length of the total DF? That would be transposition wouldn't it?

Reply:
> MEASURE,VALUE while wide basically means ID, VAR1, VAR2 with values contained in columns?

When it refers to repeated measures yes that is correct

But in general a long dataframe can also refer to long in terms of length (number of observations), which is what we wanted to say here (there are 2 cols of 4000 obs)

Reply:
Maybe it's better to call it "large" or something, I found the term "long" also confusing (it somehow implies long versus wide format).

Reply:
I'll change to lengthy in next commit?